### PR TITLE
refs sparkfabrik-innovation-team/board#1135: fix values kube-promethe…

### PR DESCRIPTION
**Pull request by filippolmt**
fix values

### CODE REVIEW

File: values/kube-prometheus-stack.yml [hunk 1]
```diff
@@ -45,7 +45,7 @@ grafana:
     storageClassName: ${storage_class_name}
     %{~ endif ~}
     size: ${grafana_pv_size}
-%{~ if prometheus_operator_resources.cpu_requests != "" && prometheus_operator_resources.memory_requests != "" ~}
+%{~ if prometheus_operator_resources.cpu_requests != "" && prometheus_operator_resources.memory_requests != "" }
```
Comment (Monska85) [inline values/kube-prometheus-stack.yml:48]:
```suggestion
%{~ if prometheus_operator_resources.cpu_requests != "" && prometheus_operator_resources.memory_requests != "" ~}
```

**Comment by Monska85** [comment]
Remove from `values/adapter-values.yml` all the default values also present [here](https://github.com/helm/charts/blob/master/stable/prometheus-adapter/values.yaml).

File: values/kube-prometheus-stack.yml (hunk 1 above)
Comment (filippolmt) [reply values/kube-prometheus-stack.yml:48]: The ~ has been voluntarily removed, if added in the generation of the values ​​"prometheusOperator" it goes up by one row configuring itself in the size of grafana, thus doing terraform apply fails

File: values/kube-prometheus-stack.yml [hunk 2]
```diff
@@ -54,8 +54,8 @@ prometheusOperator:
 %{~ endif ~}
 
 prometheus:
-  prometheusSpec:
   %{~ if prometheus_resources.cpu_requests != "" && prometheus_resources.memory_requests != "" ~}
+  prometheusSpec:
```
Comment (Monska85) [inline values/kube-prometheus-stack.yml:58]: If I understand correctly the `storageSpec` should be under the `prometheusSpec` node, so I think the old positions were correct:

```suggestion
  prometheusSpec:
  %{~ if prometheus_resources.cpu_requests != "" && prometheus_resources.memory_requests != "" ~}
```

Comment (filippolmt) [reply values/kube-prometheus-stack.yml]: Done

**Review by Monska85 — APPROVED** [review]
LGTM :+1:

**Comment by paolomainardi** [comment]
LGTM 👍

**Pull request merged**